packages/xo-web/src/xo-app/settings/servers/index.js

```diff
@@ -100,17 +100,16 @@ const COLUMNS = [
     itemRenderer: server => (
       <div>
         <StateButton
-          disabledLabel={_('serverDisconnected')}
-          disabledHandler={connectServer}
-          disabledTooltip={_('serverConnect')}
-          enabledLabel={_('serverConnected')}
-          enabledHandler={disconnectServer}
-          enabledTooltip={_('serverDisconnect')}
+          disabledLabel={_('serverDisabled')}
+          disabledHandler={enableServer}
+          disabledTooltip={_('serverEnable')}
+          enabledLabel={_('serverEnabled')}
+          enabledHandler={disableServer}
+          enabledTooltip={_('serverDisable')}
           handlerParam={server}
-          pending={server.status === 'connecting'}
-          state={server.status === 'connected'}
+          state={server.enabled === true}
         />{' '}
-        {server.error && (
+        {server.status === 'connected' && server.error != null && (
```

Comment:
Why testing the status?

Please list all cases in which we want to display a warning.

Reply:
as described in the issue #4300 
we display warining: `if enabled and disconnected, display a warning and expose the error`
And you said to me that `disconnected` means that we haven't received an event since 5 minutes

Reply:
> we display warining: `if enabled and disconnected, display a warning and expose the error`

That's not what you are testing.

And what about the already existing warning?

Reply:
so the condition should be `{server.enabled && server.error != null `